website/i18n/es.toml

```diff
@@ -0,0 +1,30 @@
+# UI strings. Buttons and similar.
+
+# Footer text
+[footer_all_rights_reserved]
+other = " | Documentación distribuida bajo CC-BY-4.0"
+
+[post_create_issue]
+other = "Crear asunto"
```

Comment:
Crear Issue. Asunto is not accurated (at least to me) on this one. In Spanish we can use freely Issue and everyone understand that.